operators/src/processing/raster_scaling.rs

```diff
@@ -185,8 +190,18 @@ where
         tile: RasterTile2D<P>,
         _pool: Arc<ThreadPool>,
     ) -> Result<RasterTile2D<P>> {
-        let offset = Self::prop_value(&self.offset, &tile.properties)?;
-        let slope = Self::prop_value(&self.slope, &tile.properties)?;
+        // either use the provided metadata/constant or the default values from the properties
```

Comment:
you could do this once in the initialization of the query processor, instead of for each tile, right?

Reply:
i guess this is a bit problematic. At least the 'Scale' and 'Offset' values might be different for different files. So we can't do this on a global time-series level. Also the data is parsed from the real GDAL datasets and that is done when the data is read. We might add information about the metadata keys to expect to the result descriptor but thats something different.